app/src/main/res/values/strings.xml

```diff
@@ -11,8 +11,7 @@
     <string name="nav_instruments">Instruments</string>
     <string name="nav_settings">Settings</string>
     <string name="nav_about_us">About Us</string>
-    <string name="nav_report">Report Issues</string>
-    <string name="nav_help_feedback">Help &amp; Feedback</string>
```

Comment:
Why remove?

Reply:
Okay @cweitat I do the changes :+1:

Reply:
@cweitat changes done :+1:  , I have renamed the section.